tests/conftest.py

```diff
@@ -38,31 +38,31 @@ def create_app():
 
 
 @pytest.fixture(scope="function")
-def superusers_role(db):
-    """Grant `superuser_access` action to the new role."""
-    superusers = Role(name="superusers")
-    db.session.add(superusers)
-    db.session.add(ActionRoles.allow(superuser_access, role=superusers))
+def admin_role(db):
+    """Grant `superuser_access` action to the new admin role."""
+    admin = Role(name="admin")
+    db.session.add(admin)
+    db.session.add(ActionRoles.allow(superuser_access, role=admin))
     db.session.commit()
-    return superusers
+    return admin
```

Comment:
Are admins and superusers the same thing? I thought someone who has access to the admin doesn't necessarily have the rights to do everything e.g., a library assistant (which has the right to access the admin panel) can come in to generate report from the admin panel and do metadata enhancements on records, but only superusers can also delete records and so on.

Reply:
It looks like indeed that there is a confusion of naming, in general. Invenio-administration has made the situation even worse.
What I want to highlight in the code is the difference between the `superuser-access` action, and any role names or users that will be allowed to perform the action. Developers are generally confused by that, and that's why I don't want to use the same name as the action.

To answer your question: it is up to the sys admin to create roles, without names that make sense in their domain. It could be `admin` or `superuser` or `librarion` or anything else.
The actions are, instead, hardcoded:
- [superusers-access](https://github.com/inveniosoftware/invenio-access/blob/master/invenio_access/permissions.py#L37)
- [administration-access](https://github.com/inveniosoftware/invenio-administration/blob/main/invenio_administration/permissions.py#L14)
You can allow any role you want to perform the actions above.

In short: it is very import to distinguish, in the code, the roles and the action. IMO, using the same naming, will create only confusion.

Reply:
No they are not: roles and actions are different, see my other comment.